internal/args/parse.go

```diff
@@ -54,8 +54,8 @@ type IgnoredConfig struct {
 	Specials        bool
 	DevicesSpecials bool   `short:"D"`
 	Times           bool   `short:"t"`
-	ATimes          bool   `short:"U"`
-	CrTimes         bool   `short:"N"`
+	ATimes          bool   `name:"atimes"  short:"U"`
```

Comment:
I suppose that `ATimes` => `Atimes` would also have solved it?

Reply:
Ah, cool. I totally missed the casing as the cause for separation.